lib/iex/lib/iex/helpers.ex

```diff
@@ -1240,11 +1240,24 @@ defmodule IEx.Helpers do
 
       iex> pid("0.21.32")
       #PID<0.21.32>
+      
+      iex> pid("#PID<0.21.32>")
+      #PID<0.21.32>
 
       iex> pid(:init)
       #PID<0.0.0>
 
   """
+  def pid("#PID<" <> _rest = string) do
+    case Regex.named_captures(~r/#PID<(?<triplet>\d+.\d+.\d+)>/, string) do
```

Comment:
I don’t think we need the regex. We can strip the #PID part, convert the rest to char list and given to Erlang. Erlang will raised if malformed.